CHANGELOG.md

```diff
@@ -76,6 +76,8 @@ The changes are relative to the previous release, unless the baseline is specifi
 * android_jni: Set threads to 2 instead of CPU count
 * Fix overflows when dealing with alpha during YUV/RGB conversions and in
   avifRGBImageAllocatePixels().
+* Make avifEncoder.headerFormat a flag combination for future features.
+* Rename AVIF_HEADER_FULL to AVIF_HEADER_DEFAULT. Deprecate AVIF_HEADER_FULL.
```

Comment:
I removed the v1.2.0 milestone from this pull request. This pull request doesn't need to go into the libavif v1.2.0 release. This will give us more time to work out a good solution.